### PR TITLE
Add listen option to the configuration file of graphkb.

### PR DESCRIPTION
**Pull request by clems4ever**

### CODE REVIEW

File: internal/server/server.go [hunk 1]
```diff
@@ -379,17 +375,16 @@ func StartServer(database knowledge.GraphDB, schemaPersistor schema.Persistor,
 	r.HandleFunc("/api/query", postQueryHandler).Methods("POST")
 	r.PathPrefix("/").Handler(http.FileServer(http.Dir("./web/build/")))
 
-	bindInterface := fmt.Sprintf(":%d", viper.GetInt32("port"))
-	fmt.Printf("Listening on %s\n", bindInterface)
+	fmt.Printf("Listening on %s\n", listenInterface)
```
Comment (pierresouchay) [inline internal/server/server.go]: I would write a single message a bit later combined with TLS information

File: internal/server/server.go [hunk 2]
```diff
@@ -379,17 +375,16 @@ func StartServer(database knowledge.GraphDB, schemaPersistor schema.Persistor,
...
 
 	var err error
 	if viper.GetString("tls_cert") != "" {
 		fmt.Println("Server is using TLS, the connection is secure")
-		err = http.ListenAndServeTLS(bindInterface, viper.GetString("tls_cert"),
+		err = http.ListenAndServeTLS(listenInterface, viper.GetString("tls_cert"),
```
Comment (pierresouchay) [inline internal/server/server.go]: If would push the message here:
```
fmt.Printf("Listening on %s with TLS enabled\n", listenInterface)
```

File: internal/server/server.go [hunk 3]
```diff
@@ -379,17 +375,16 @@ func StartServer(database knowledge.GraphDB, schemaPersistor schema.Persistor,
...
 			viper.GetString("tls_key"), r)
 	} else {
 		fmt.Println("[WARNING] Server is NOT using TLS, the connection is not secure")
```
Comment (pierresouchay) [inline internal/server/server.go]: Here I would put the messge with how fixing it:
```
fmt.Printf("[WARNING] Listening on %s without TLS, use `tls_cert` option to setup a certificate\n", listenInterface)
```

File: internal/server/server.go (hunk 1 above)
Comment (clems4ever) [reply internal/server/server.go]: +1

File: internal/server/server.go (hunk 3 above)
Comment (clems4ever) [reply internal/server/server.go]: +1

**Pull request merged**